packages/@aws-cdk/aws-servicecatalogappregistry/test/integ.application-associator.cross-account-stack-association-enabled.ts

```diff
@@ -0,0 +1,22 @@
+import * as cdk from '@aws-cdk/core';
+import * as integ from '@aws-cdk/integ-tests';
+import * as appreg from '../lib';
+
+const app = new cdk.App();
+const localStack = new cdk.Stack(app, 'integ-servicecatalogappregistry-first-resource');
+
+new appreg.ApplicationAssociator(app, 'RegisterCdkApplication', {
```

Comment:
the expected way to do these tests is to use your personal account IDs when you deploy it, but switch them to dummy account IDs when you submit the PR.

Reply:
Understood; as part of this next commit, I took these steps:

1.  Specified `env` and `account` properties such that `localStack` and the Application stack was in Account 1, and `crossAccountStack` was in Account 2
2. Set up Accounts 1 and 2 such that they were both in the same AWS organization with cross-account sharing enabled
3. Modified Account 2's CDK role(s) to allow Account 1 to perform CDK deployments
4. Ran `yarn integ --update-on-failed` 

After successful deployments of all stacks in their respective accounts which generated the test snapshots, I replaced all mentions of Accounts 1 and 2 with `${AWS::AccountId}` in the snapshots and removed the `env` properties in this test file.